pages/orgs/[orgId]/apps/[appId]/messages/new.tsx

```diff
@@ -128,200 +134,202 @@ export default function NewMessageForAppPage() {
-                      return (
-                        <TableRow key={index}>
-                          <TableCell>
-                            <Select
-                              label="ButtonDesign"
-                              value={action.buttonDesign}
-                              onChange={(event) =>
-                                handleButtonDesignChange(index, event)
-                              }
-                            >
-                              {buttonDesigns.map((value, index) => {
-                                return (
-                                  <MenuItem key={index} value={value}>
-                                    {value}
-                                  </MenuItem>
-                                );
-                              })}
-                            </Select>
-                          </TableCell>
-                          <TableCell>
-                            <Select
-                              label="ActionType"
-                              value={action.actionType}
-                              onChange={(event) =>
-                                handleActionTypeChange(index, event)
-                              }
-                            >
-                              {actionTypes.map((value, index) => {
-                                return (
-                                  <MenuItem key={index} value={value}>
-                                    {value}
-                                  </MenuItem>
-                                );
-                              })}
-                            </Select>
-                          </TableCell>
-                          <TableCell>
-                            <TextField
-                              type="text"
-                              name="actionTitle"
-                              value={action.title}
-                              onChange={(event) =>
-                                handleActionTitleChange(index, event)
-                              }
-                            />
-                          </TableCell>
-                          <TableCell>
-                            <IconButton onClick={() => deleteAction(index)}>
-                              <MdDeleteForever />
-                            </IconButton>
-                          </TableCell>
-                        </TableRow>
-                      );
-                    })}
-                </TableBody>
-              </Table>
-              {actions.length == 0 && (
-                <p className="marginTopSmall centeredElement">
-                  no actions added
-                </p>
-              )}
-              <div className="addButton centeredElement">
-                <Button
-                  variant="contained"
-                  onClick={() => {
-                    addAction();
-                  }}
+        <div className="flex flex-row pt-8 px-8 justify-center">
```

Comment:
Any reason not to use the Flex components of ChakraUI?

Reply:
no, i didnt give it much thought, because there would be a lot of changes anyway when Jules finishes the design